### PR TITLE
#1672 Add a link back to workspace's topics tab in pipeline's new topic modal

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1672)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/pages/PipelinePage/PipelineToolbar/PipelineNewTopic.js [hunk 1]
```diff
@@ -68,22 +73,29 @@ class PipelineNewTopic extends React.Component {
...
   render() {
-    const { topics, currentTopic } = this.props;
-
-    if (!topics) return null;
+    const { topics, currentTopic, workerClusterName: workspace } = this.props;
 
     return (
-      <Box shadow={false}>
-        <React.Fragment>
+      <Wrapper>
```
Comment (wu87988622) [inline ohara-manager/client/src/components/pages/PipelinePage/PipelineToolbar/PipelineNewTopic.js]: Can you add some function to pipeline page streamApp ? thanks.

Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/PipelineToolbar/PipelineNewTopic.js]: I will file another issue for that :)

Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/PipelineToolbar/PipelineNewTopic.js]: Hmm, maybe in this issue as it's not a big change anyway...

Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/PipelineToolbar/PipelineNewTopic.js]: A quick look a the source code made me think about splitting the issue as there is more work than I was anticipating. The stream app part will be tackle in #1687

**Comment by konekoya** [comment]
retry manager-it

**Pull request merged**